src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="images of imac"
+        class="card-image"
+      >
+      <div class="text-container">
+        <div class="card-text">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+        <div class="card-product-Code">
+          Код товара: 195434
+        </div>
+        <div class="container-feedback">
+          <div class="star-stars">
+            <div class="stars"></div>
+            <div class="stars"></div>
+            <div class="stars"></div>
+            <div class="stars"></div>
+            <div class="stars"></div>
+          </div>
+          <div class="feedback">
```

Comment:
```suggestion
          <div class="card__feedback">
```
don't make a new BEM-block for each line of text

Reply:
fix it everywhere